Source/santametricservice/Writers/SNTMetricHTTPWriter.m

```diff
@@ -72,30 +71,28 @@ - (BOOL)write:(NSArray<NSData *> *)metrics toURL:(NSURL *)url error:(NSError **)
                           [NSString stringWithFormat:@"received http status code %ld from %@",
                                                      httpResponse.statusCode, url]
                       }];
-              *stop = YES;
             }
           }
           dispatch_group_leave(requests);
         }];
 
     [task resume];
+  }
 
-    SNTConfigurator *config = [SNTConfigurator configurator];
-    int64_t timeout = (int64_t)config.metricExportTimeout;
+  SNTConfigurator *config = [SNTConfigurator configurator];
```

Comment:
Minor: Could we move this to the top of the function, something like `static SNTConfigurator *config = [SNTConfigurator configurator];`. Even though this code won't be hit that often, we've seen in measurements that this lookup takes a surprising amount of time, would be good to eliminate it when sending metrics.